libnavigator/src/main/java/com/mapbox/navigation/navigator/internal/MapboxNativeNavigator.kt

```diff
@@ -111,7 +111,7 @@ interface MapboxNativeNavigator {
      *
      * @return True if the annotations could be updated false if not (wrong number of annotations)
      */
-    fun updateAnnotations(legAnnotationJson: String, routeIndex: Int, legIndex: Int): Boolean
+    fun updateAnnotations(legAnnotationJson: String, legIndex: Int): Boolean
```

Comment:
for now the SDK supports single route only. Made it consistent with `fun setRoute`